CHANGELOG.md

```diff
@@ -1,3 +1,37 @@
+# [MongooseIM 6.3.3](https://github.com/esl/MongooseIM/releases/tag/6.3.3) - 2025-04-11
+
+## Highlights
+- Fixed buffer duplication issue in stream management
+- Improved IQ handling for unknown ID formats
+- Restored package support for older systems
+- Various enhancements and bug fixes
+
+## Changed
+- Minor refactor of `mod_stream_management` (#4497)
```

Comment:
Maybe move it to "Other", because there is no change from the user's point of view?